.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
I suggest adding `.idea` to your `.gitignore` file or your `.gitignore_global` file.